dir.props

```diff
@@ -22,7 +22,7 @@
     <!-- Temporarily omit code analysis -->
     <SkipImportCodeAnalysisTargets>true</SkipImportCodeAnalysisTargets>
     <!-- Hacky BuildConfiguration -->
-    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp1.1-$(OSEnvironment)</BuildConfiguration>
+    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp1.1-$(OSEnvironment)-x64-Debug</BuildConfiguration>
```

Comment:
This should be "-Debug-x64" for consistency with the spec. I suspect what you have will actually still work but I would like to keep them consistent.